zeq/event.h

```diff
@@ -35,7 +38,15 @@ class Event : public boost::noncopyable
     ZEQ_API explicit Event( const uint128_t& type );
 
     /** Move ctor @internal */
+#ifndef BOOST_NO_CXX11_RVALUE_REFERENCES
     ZEQ_API Event( Event&& rhs );
+#else
+    ZEQ_API Event( const Event& rhs )
+    {
+        _impl = rhs._impl;
+        const_cast<Event &>(rhs)._impl = 0;
+    }
```

Comment:
If possible, I'ld rather make rhs non-const and remove the const-cast. This way it is clear in the API.